schema/mysql-upgrades/2.12.0.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE icingaweb_schema
```

Comment:
I was able to find only documents up to [MYSQL >= `5.7`]( https://dev.mysql.com/doc/refman/5.7/en/drop-table.html).

Reply:
I think the exists extensions for `DROP CONSTRAINT/INDEX` and `ADD COLUMN` don't work. See [here](https://downloads.mysql.com/docs/refman-5.6-en.pdf). Idempotent migrations would only be possible with something like [this](https://github.com/Icinga/icingaweb2/blob/feature/optimize-ido-mysql/modules/monitoring/schema/migration-procedures.sql).

Reply:
> Please note that "IF EXISTS" is only available in MySQL 5.1.0 or later.

I know this is not the official documentation, but according to this [website](https://www.w3docs.com/snippets/php/how-can-i-employ-if-exists-for-creating-or-dropping-an-index-in-mysql.html ) the `IF EXISTS` is supported since `5.1`.

Reply:
Please test this in a MySQL 5.5 docker container and share the results.

Reply:
Even MYSQL 5.7 doesn't fully support the `IF EXISTS' extension, so I'm going to drop it from here and x509/reporting modules.

Reply:
Yes, I would not try to make the scripts idempotent at the moment.